conformance/google-java/src/test/kotlin/com/connectrpc/conformance/Conformance.kt

```diff
@@ -92,11 +94,28 @@ class Conformance(
                 "--key",
                 "cert/localhost.key"
             )
+            .waitingFor(HostPortWaitStrategy().forPorts(8081))
+
+        @JvmField
+        @ClassRule
+        val CONFORMANCE_CONTAINER_GRPC = GenericContainer("connectrpc/conformance:$CONFORMANCE_VERSION")
+            .withExposedPorts(8081)
+            .withCommand(
+                "/usr/local/bin/servergrpc",
+                "--port",
+                "8081",
+                "--cert",
+                "cert/localhost.crt",
+                "--key",
+                "cert/localhost.key"
+            )
+            .waitingFor(HostPortWaitStrategy().forPorts(8081))
     }
 
     @Before
     fun before() {
-        val host = "https://localhost:${CONFORMANCE_CONTAINER.getMappedPort(8081)}"
+        val serverPort = if (serverType == ServerType.CONNECT_GO) CONFORMANCE_CONTAINER_CONNECT.getMappedPort(8081) else CONFORMANCE_CONTAINER_GRPC.getMappedPort(8081)
```

Comment:
Not a big deal, but calling out that Swift uses 8083 for grpc-go because that's where it's hosted in the container: https://github.com/connectrpc/connect-swift/blob/f7aab0e53c38f15d5ab62e8e8afa2f984c7e34f3/Tests/ConnectLibraryTests/ConnectConformance/ConformanceConfiguration.swift#L58